assets/large_language_models/rag/environments/rag_embeddings/context/conda_dependencies.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - azure-ai-ml=={{latest-pypi-version}}
     # Azure
   - azure-core<2.0.0,>=1.8.0,!=1.22.0
-  - azure-identity==1.12.0
+  - azure-identity=={{latest-pypi-version}}
```

Comment:
Is this value templated in? If so, how do we deal with the case where azure-ai-ml and azure-identity have different versions?

Reply:
Yes. These version is filled by build process with the latest version on PyPi at build time.

If there is version dependency issue, the build will break, then we can fix the build break by pinning the versions.